akka-http/src/main/scala/akka/http/scaladsl/coding/Deflate.scala

```diff
@@ -102,6 +105,7 @@ class DeflateDecompressor(maxBytesPerChunk: Int = Decoder.MaxBytesPerChunkDefaul
   }
 }
 
+@deprecated("Use `akka.stream.scaladsl.Compression.inflate` instead", "3.0.0-RC2")
```

Comment:
There's no alternative for this one. It should have been private from the beginning.